src/stories/Card/Card.tsx

```diff
@@ -1,28 +1,51 @@
 import './card.scss'
+import { Image } from '../Image/Image'
+import { ReactElement } from 'react'
 
 interface CardProps {
-    title?: string
-    type?: 'condensed' | 'spacious' | 'blue' | 'danger'
-    background?: string
-    children?: any
+    img?: boolean
+    img_src?: any
```

Comment:
We should condense this down to 

If an `img_src` is provided, we show an image
If not, we don't